agenta-backend/agenta_backend/models/api/evaluation_model.py

```diff
@@ -293,9 +293,15 @@ class NewEvaluation(BaseModel):
     evaluators_configs: List[str]
     testset_id: str
     rate_limit: LLMRunRateLimit
-    lm_providers_keys: Optional[Dict[LMProvidersEnum, str]] = None
+    lm_providers_keys: Optional[Dict[str, str]] = None
```

Comment:
this looks like it's coming from the vault PR, no @aakrem @aybruhm

Reply:
Nope, from the supertokens fix.